padl/util_transforms.py

```diff
@@ -108,3 +108,51 @@ class IfTrain(IfInMode):
 
     def __init__(self, if_: Transform, else_: Optional[Transform] = None):
         super().__init__(if_, 'train', else_)
+
+
+class Try(ClassTransform):
+    """ Perform *transform*. If this fails with any exception from *exceptions*, perform
+    *catch_transform*.
+
+    :param transform: Transform to try
+    :param catch_transform: Transform to fall back on
+    :param exceptions: Catch conditions
+    """
+
+    def __init__(self,
+                 transform: Transform,
+                 catch_transform: Transform,
+                 exceptions: Union[List, Tuple, Exception],
+                 else_transform: Transform = Identity(),
+                 finally_transform: Transform = Identity(),
+                 pd_name: str = None):
+
+        if not isinstance(exceptions, (tuple, list)):
+            exceptions = (exceptions, )
+        exceptions = tuple(exceptions)
+        for exception in exceptions:
+            assert issubclass(exception, Exception)
+        super().__init__(pd_name=pd_name,
+                         arguments=OrderedDict([('transform', transform),
+                                                ('catch_transform', catch_transform),
+                                                ('exceptions', exceptions)]))
```

Comment:
Should the `else_transform` and `finally_transform` be added to the `OrderedDict`?